code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm

```diff
@@ -63,7 +63,7 @@
 /datum/action/xeno_action/activable/gravity_crush/use_ability(atom/A)
 	owner.face_atom(A) //Face towards the target so we don't look silly
 	var/list/turfs = RANGE_TURFS(1, A)
-	playsound(A, 'sound/effects/bomb_fall.ogg', 75, FALSE)
+	playsound(A, 'sound/effects/bomb_fall.ogg', 50, FALSE)
```

Comment:
this doesn't do anything, volume is bugged. You have to edit the file yourself

Reply:
Since when is it bugged?